### ISSUE TITLE
[BUG] Games crashed 

### ISSUE THREAD

**Issue by Kingbb1** [longjunyu2/winlator#42]
**Device Information**
- Device Model: [Samsung Galaxy S24ultra]
- Device Processor Model: [Snapdragon 8gen3]
- Android Version: [14]

**Application Configuration Information**
- App Version: [7.1.1/7.1.2]
- Display Driver: [Turnip-24.3.0]
- dxwrapper: [DXVK-2.3.1]
- box64 Version: [0.3.0]
- box64 Preset: [performance]

Hi longjunyu2 First Thanks for work hard for this emu I appreciate your work ❤️

I have some problems 
Resident evil 2 Remake 
And Resident evil 3 Remake 
It crashes when entering gameplay. I tried all the settings, changing Turnip, Dxvk, box64, and changing box64 Preset. I tried all of them, the same problem. I tried the old version of winlator glibc 7.1.1. 
And new
winlator glibc 7.1.2 
But I saw videos that played most of the games that don't work for me, but their processors are Snapdragon 8gen 2 or lower. Is this because of my newer processor? 
I tried winlator afei mod and it all worked, but I hope it will work in winlator glibc and fix most of the problems. Thank you very much ❤️

**Comment by longjunyu2** [comment]
I'm not quite sure why this happened. These version contains some new things (new wine and new box64, maybe faster, maybe unstable?).
Perhaps others can help you, good luck!

**Comment by Kingbb1** [comment]
> لست متأكدًا تمامًا من سبب حدوث ذلك. تحتوي هذه الإصدارات على بعض الأشياء الجديدة (إصدار جديد من wine وbox64، ربما يكون أسرع، أو ربما غير مستقر؟). ربما يمكن للآخرين مساعدتك، حظًا سعيدًا!

**Comment by Kingbb1** [comment]
Yes I did. I took the settings from people I used the emulator I tried all the settings that worked with them, but they did not work for me, so I did not know why. What's the problem? I tried using .
Winlator afei mod 
Original version last modified 
not Winlator glibc afei mod. So All the games I told you worked on it, but I was hoping it would work in winlator glibc because the performance in it is fast and I liked it, thank you for reply i appreciate your work ❤️🫡

**Comment by Saikatsaha1996** [comment]
> **Device Information**
> 
> * Device Model: [Samsung Galaxy S24ultra]
> * Device Processor Model: [Snapdragon 8gen3]
> * Android Version: [14]
> 
> **Application Configuration Information**
> 
> * App Version: [7.1.1/7.1.2]
> * Display Driver: [Turnip-24.3.0]
> * dxwrapper: [DXVK-2.3.1]
> * box64 Version: [0.3.0]
> * box64 Preset: [performance]
> 
> Hi longjunyu2 First Thanks for work hard for this emu I appreciate your work ❤️
> 
> I have some problems Resident evil 2 Remake And Resident evil 3 Remake It crashes when entering gameplay. I tried all the settings, changing Turnip, Dxvk, box64, and changing box64 Preset. I tried all of them, the same problem. I tried the old version of winlator glibc 7.1.1. And new winlator glibc 7.1.2 But I saw videos that played most of the games that don't work for me, but their processors are Snapdragon 8gen 2 or lower. Is this because of my newer processor? I tried winlator afei mod and it all worked, but I hope it will work in winlator glibc and fix most of the problems. Thank you very much ❤️

@Kingbb1 what is your wine version?

**Comment by Kingbb1** [comment]
> > **معلومات الجهاز**
> > 
> > * طراز الجهاز: [Samsung Galaxy S24ultra]
> > * طراز معالج الجهاز: [Snapdragon 8gen3]
> > * إصدار أندرويد: [14]
> > 
> > **معلومات تكوين التطبيق**
> > 
> > * إصدار التطبيق: [7.1.1/7.1.2]
> > * برنامج تشغيل العرض: [Turnip-24.3.0]
> > * dxwrapper: [DXVK-2.3.1]
> > * إصدار box64: [0.3.0]
> > * box64 الإعداد المسبق: [الأداء]
> > 
> > مرحبًا longjunyu2، أولًا، شكرًا لك على العمل الجاد من أجل هذا الإيمو، أقدر عملك ❤️
> > عندي بعض المشاكل في Resident evil 2 Remake و Resident evil 3 Remake تتعطل اللعبة عند الدخول إلى اللعبة. جربت كل الإعدادات، تغيير Turnip و Dxvk و box64 وتغيير الإعداد المسبق box64. جربتهم جميعًا، نفس المشكلة. جربت الإصدار القديم من winlator glibc 7.1.1. و winlator glibc 7.1.2 الجديد ولكنني رأيت مقاطع فيديو لعبت معظم الألعاب التي لا تعمل معي، لكن معالجاتها هي Snapdragon 8gen 2 أو أقل. هل هذا بسبب معالجي الأحدث؟ جربت winlator afei mod ونجح كل شيء، لكن أتمنى أن يعمل في winlator glibc ويصلح معظم المشاكل. شكرا جزيلا ❤️
> 
> @Kingbb1ما هو إصدار النبيذ الخاص بك؟

**Comment by Kingbb1** [comment]
@Saikatsaha1996 Saikatsaha1996

Hi, I use wine 9.2 costem. Are you saying this is a problem with wine-9.2-custom? Did I work with you? Because the same wine is used in winlator afei mod, the original modified version, the latest version, and I worked on it
 Resident Evil 2 Remake 
And Resident Evil 3 Remake 
And winlator glibc afei mod 
It didn't work, although it's the same wine used in the emulators. I don't know what the problem is 👀?

**Comment by Saikatsaha1996** [comment]
> @Saikatsaha1996 Saikatsaha1996
> 
> Hi, I use wine 9.2 costem. Are you saying this is a problem with wine-9.2-custom? Did I work with you? Because the same wine is used in winlator afei mod, the original modified version, the latest version, and I worked on it Resident Evil 2 Remake And Resident Evil 3 Remake And winlator glibc afei mod It didn't work, although it's the same wine used in the emulators. I don't know what the problem is 👀?

No 9.2 working fine for me

**Comment by Kingbb1** [comment]
@longjunyu2 bro I thank you very much, you and Matrin, who contributed to a project that fixed the problem and worked ❤️ for me
Resident Evil 2 Remake and 
Resident evil 3 Remake 
Update Winlator glibc 7.1.3
But I have a problem in Resident Evil 2 Remake. The control moves from a fixed state. What is the solution to the problem?

**Comment by Kingbb1** [comment]
bro @longjunyu2 Thank you very much and whoever contributed to this project. It fixed the problem of automatic exit from the game that I had in
 Resident Evil 2 Remake and
Resident evil 3 Remake 
In winlator glibc 7.1.3 update ❤️
But I have a problem in Resident Evil 2 Remake. The controller or joystick moves automatically. I do not move it myself in the controller or when touching the screen. How do I fix this problem?

**Comment by longjunyu2** [comment]
> bro @longjunyu2 Thank you very much and whoever contributed to this project. It fixed the problem of automatic exit from the game that I had in Resident Evil 2 Remake and Resident evil 3 Remake In winlator glibc 7.1.3 update ❤️ But I have a problem in Resident Evil 2 Remake. The controller or joystick moves automatically. I do not move it myself in the controller or when touching the screen. How do I fix this problem?

Go Edit Container -> Advanced, enable xinput and disabled dinput.

**Comment by longjunyu2** [comment]
@Kingbb1 ok, that's greate. But did you try my method before it?

**Comment by Kingbb1** [comment]
> @Kingbb1انا، هذا رائع. ولكن هل جربت طريقتي من قبل؟

Yeah but not working 


This is someone who wanted to fix the problem and clarify some points 👇

Listen, these are my own Dinput files. They are only suitable for games that support Dinput in my game files, because when I added them to my own game that did not work with them, these are my Dinput files because the game basically does not support Dinput in its own files. This is what led to me ruining the game. It no longer works for me when I want to start playing it

When I added the Dinput files to the game, the game started to freeze when I wanted to start playing, and then a private message appears: “Wine” and all this is because of the library when I added Dinput to it 😂
These files fix some games that support Dinput and break games that do not support Dinput  😃

I don't know of some people who tried it and it worked for them and fixed the control problem, including me, but this person did the same method, but it did not work for him, so these points may help you 👍

This is another clarification from> klark Gaming Channel 👇

Its not xinput to xinput bridge. its actually xinput to dinput. it emulates dinput. I just use it before for xinput bridge for Resident Evil 7.

And Thank you so much for your reply and For work hard for that emu ❤️

**Comment by WaspPC** [comment]
I have a similar issue.

I have the UK Edition of the S24Ultra ARM64, trying to run Serious Sam 2.

The S24Ultra has 12gb ram with 8gb virtual memory, but Winulator says only 4gb is available, but the DX9 gfx mode causes a fatal error and crash.

Containers are as guided, Serious Sam 2 Setup is as guided by user reference that has had it working, but i keep getting "cannot set display mode" and "cannot create rendering device", click ok and crash to homescreen.
